cgogn/ui/imgui_helpers.h

```diff
@@ -43,63 +43,32 @@ namespace ui
  * @brief generate combo box for attribute selection
  * @param m mesh
  * @param label label of the combo box
- * @param selected_attribute current selected attribute
+ * @param selected_attribute current selected attribute (raw or shared pointer)
```

Comment:
For now; if unique/weak pointers are also added to the allowed types, then mention them here too